src/main/java/seedu/address/logic/commands/UpdateCommand.java

```diff
@@ -167,4 +178,10 @@ public String toString() {
                 .add("removeIndices", removeIndices)
                 .toString();
     }
+
+    private void checkStartDateBeforeEnd(LocalDate sd, LocalDate ed) throws CommandException {
+        if (sd != null && ed != null && sd.compareTo(ed) > 0) {
```

Comment:
Maybe we could use `requireAllNonNull` method, as we are using it in many other places of the code base. Helps with SLAP principle.

Reply:
Also, instead of using compareTo, LocalDate has built in methods to check whether a LocalDate is before/after/same as another LocalDate.